jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -427,7 +427,7 @@ public Request newRequest(String uri)
      */
     public Request newRequest(URI uri)
     {
-        return newHttpRequest(newConversation(), uri);
+        return newHttpRequest(newConversation(), checkHostInUri(uri));
```

Comment:
There is no need to call `checkHostInUri()` here, since it will be called again in the chained method. Just forward the URI parameter as it was before.